src/editor/nodes/SceneNode.js

```diff
@@ -637,7 +637,7 @@ export default class SceneNode extends EditorNodeMixin(Scene) {
       if (!attribution) return;
 
       if (attribution) {
-        const attributionKey = attribution.url || `${attribution.name}_${attribution.author}`;
```

Comment:
`getContentAttributions` is only gathering attributions for the `Model` nodes in the scene. We should check the `attribution` property instead of an explicit node type to see if it should be included.

Reply:
Oops, missed that. Fixed.